src/components/LoadFileMenu/index.tsx

```diff
@@ -117,18 +117,14 @@ const LoadFileMenu = ({
 
     return (
         <>
-            <Dropdown
-                menu={{ items, theme: "dark", className: styles.menu }}
```

Comment:
Do we lose the dark theming here on the dropdown?

Reply:
No we don't, although we are overriding most of it anyway.